debian/rules

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_clean:
+	dh_clean
+	rm -f CMakeCache.txt Makefile cmake_install.cmake haisrt.pc install_manifest.txt srt.pc
+	rm -f libsrt.so.1.2.0 stransmit utility-test libsrt.so.1 libsrt.so
```

Comment:
libsrt.so.1.3.3
`stransmit` is no longer present. Now there are `srt-live-transmit` and `srt-file-transmit`.

Reply:
that `rm -f` feels wrong, doesn't the cmake file generate a working "make clean"? If not, it should be fixed to do so.